packages/generators/src/addon-generator.ts

```diff
@@ -55,7 +55,8 @@ const addonGenerator = (
                 try {
                     mkdirp.sync(pathToProjectDir);
                 } catch (err) {
-                    logger.error('Failed to create directory', err);
+                    logger.error('Failed to create directory');
```

Comment:
logger.error(`Failed to create directory: ${err}`);

Reply:
See comment above about `error.toString()`

Reply:
No:

**error.js**

```js
const error = new Error('test');

console.log(error);
```

and run `node: error.js`

```
Error: test
    at Object.<anonymous> (/home/path/to/test.js:1:77)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)

```

Reply:
This is special handling by the console functions.

The code doesn't do that because `logger.error` does this:
```
console.log(`[webpack-cli] ${error}`)
```

Which at runtime is the same as:
```js
console.log('[webpack-cli] ' + error.toString())
```

Which does **not** print the stack trace. Did you check out the test repo which showcases the behavior?

Reply:
A more direct test would be to log the error using the logger. You can create a `test.js` file in the root of the repo with this code:
```js
const logger = require("webpack-cli/lib/utils/logger");
logger.error(new Error("test"));
```

And see it not have the expected behavior when you run it with `node test.js`